provider/aws/formation/resource/postgres.json.tmpl

```diff
@@ -5,10 +5,18 @@
     "Version9": { "Fn::Equals": [ { "Fn::Select": [ 0, { "Fn::Split": [ ".", { "Ref": "Version" } ] } ] }, "9" ] }
   },
   "Parameters": {
+    "BackupRetentionPeriod": {
+      "Type": "String",
+      "Default": "1"
+    },
     "Class": {
       "Type": "String",
       "Default": "db.t2.micro"
     },
+    "Database": {
```

Comment:
Is there any real advantage to parameterizing this since the result comes out in the URL?

Reply:
We have some apps that we will dump data into, which have DB names already declared. This is for apps that we migrating and may have external clients that also connect to DBs.